### DISCUSSION TITLE
FluentTextField mask first 5 characters

### DISCUSSION THREAD

**Discussion by Nehakrg** [microsoft/fluentui-blazor#1789] [Q&A] [answered]
Is there a way to mask the first 5 characters as shown in the placeholder? 
<FluentTextField   @bind-Value="user.SocialSecurityNum" Maxlength="9" Minlength="9" Placeholder="*****####" />

How to use the pattern parameter to display the phone number with dashes?
<FluentTextField  Id="txtCellPhoneNo" @bind-Value="user.PhoneNum" Placeholder="###-###-####" InputMode="InputMode.Telephone" Minlength="10" Maxlength="10"/>

**Comment by vnbaaij** [comment, marked as answer]
> Is there a way to mask the first 5 characters as shown in the placeholder? <FluentTextField @bind-Value="user.SocialSecurityNum" Maxlength="9" Minlength="9" Placeholder="*****####" />

No, we do not have any functionality for that.  Possible solution might be to subclass this component in a component of your own (see https://chrissainty.com/building-custom-input-components-for-blazor-using-inputbase/ for some more information)

> How to use the pattern parameter to display the phone number with dashes? <FluentTextField Id="txtCellPhoneNo" @bind-Value="user.PhoneNum" Placeholder="###-###-####" InputMode="InputMode.Telephone" Minlength="10" Maxlength="10"/>
The `Pattern` parameter can only be used to check if the inputted text follows the pattern.

**Reply by Nehakrg** [reply]
Thank you.
